contributing.md

```diff
@@ -1,9 +1,109 @@
+
+## Ways to Contribute
+
+### Reporting Bugs
+
+If you happen to run into a bug, please post an Issue on our main GitHub Issue board:
+https://github.com/directus/directus/issues
+
+Please be as explicit and detailed as you can in the bug report. The more information is available, the easier it is for
+other contributors to help you find the solution or fix.
+
+### Leaving Feedback
+
+If you have a great idea for an improvement of the platform, or any other feedback, please make sure to open a new
+Discussion on our GitHub Discussions board: https://github.com/directus/directus/discussions
+
+### Document the Project
+
+[The Directus Docs](https://github.com/directus/docs) are a living document that can always be improved on. Any parts of
+the docs in dire need of some tender love and care? Feel free to open a Pull Request!
+
+### Helping Others
+
+The Directus community is growing quickly, which also means there's more and more people that have questions. Helping
+out your fellow developers by answering questions on [Discord](https://directus.chat) or
+[GitHub Discussions](https://github.com/directus/directus/discussions/categories/q-a) is a great way to help the
 project.
 
-Learn more about contributing to the platform: https://docs.directus.io/contributing/introduction/
+### Pull Requests
+
+#### Bug Fixes
+
+We treat Issues on the main repo as actionable items we want to get done. This also means that we welcome PRs for any
+Issue that has been labeled. Labeled issues are bugs of new features that have been triaged, accepted, and are ready to
+be implemented.
+
+#### Implementing Features
+
+With the continuous growth of Directus, more and more people are relying on Directus for (critical) data workloads in
+various use cases. This means we need to be careful with any changes that might affect the stability, security,
+performance, or scalability of Directus. For this reason, it's important that any new feature is properly thought
+through and discussed before being implemented.
+
+Before you start writing code to implement your new feature idea, please read through and understand our triaging
+process for new features before diving in. While we encourage and appreciate every code contribution, please understand
+that we can't merge every suggested code change.
+
+##### Triaging Process
+
+Feature Request Discussions that are deemed ready to be implemented with the discussed implementation details are marked
+"Accepted" and converted into an Issue, at which point the feature is ready to be implemented.
+
+New feature ideas reported directly to issues might be converted into a Discussion for further triaging at
+[the core team](https://github.com/orgs/directus/people)'s discretion first. This is often due to a lack of detail, or
+lack of proven interest.
+
+Each Pull Request that comes in is required to resolve [an open Issue](https://github.com/directus/directus/issues).
```

Comment:
Although the "Bug Fixes" section already highlighted the issue needs to be labeled/accepted, but should we explicitly (re)mention `labeled`/`accepted` in `an open Issue`?